Source/items.h

```diff
@@ -17,6 +17,100 @@ enum item_quality {
 	ITEM_QUALITY_UNIQUE,
 };
 
+enum _unique_items : int8_t {

```

Comment:
This being plural really confuses me. In .Net, plural enums are reserved to flags, while singular ones are standard sequential ones.

Reply:
enum names are a mess, but as mentioned previously, im the wrong person for correcting that.